### ISSUE TITLE
Refactor: Alerts in application to working good with Next.js + Tailwind

### ISSUE THREAD

**Issue by heronlancellot** [w3b3d3v/web3task-front#145]
Proposal:
- We currently use those imports and the **handleSnackbar** to handle all the alerts. We should move that to _Shadcn_ Alerts with animation like the older.

```
import Snackbar from '@mui/material/Snackbar';
import MuiAlert, { AlertColor, AlertProps } from '@mui/material/Alert';
```
- [ ] Remove those currently imports & Alerts in application
- [ ] Create this Alert with [shadcn](https://ui.shadcn.com/)

Or suggest a better one way!

### _Attention_ 
This task should be pushed to the **Next.js** _refactoring_ **branch** #16

**Comment by diegosano** [comment]
@heronlancellot hey, can I work on this one?

**Comment by heronlancellot** [comment]
yupp, assigned to you @diegosano  !

**Comment by heronlancellot** [comment]
closed by #160
